apps/api/src/app/shared/shared.module.ts

```diff
@@ -95,6 +98,9 @@ const PROVIDERS = [
   InvalidateCacheService,
   storageService,
   ...DAL_MODELS,
+  ExecutionLogRoute,
+  CreateExecutionDetails,
+  getIsExecutionLogQueueEnabled,
```

Comment:
Those are here because of ConditionsFilter deps we have in IntegrationModule. i believe if we were able to merge Pablo's PR regarding the filtering service this part would not be needed.